pkg/controller/namespace/namespace_controller.go

```diff
@@ -132,7 +132,9 @@ func (nm *NamespaceController) enqueueNamespace(obj interface{}) {
 		glog.Errorf("Couldn't get key for object %+v: %v", obj, err)
 		return
 	}
-	nm.queue.Add(key)
+	// delay processing namespace deletions long enough for the lifecycle admission
+	// plugins to also observe the deletion and stop new objects from being created
+	nm.queue.AddAfter(key, 5*time.Second)
```

Comment:
@derekwaynecarr I think we need this in both HA master and HA etcd scenarios, including when the master is behind a load balancer and the endpoints only list a single IP. Not sure about the duration